### ISSUE TITLE
Gapseq can't be installed anymore because `sybil` has been removed from CRAN

### ISSUE THREAD

**Issue by cdiener** [jotech/gapseq#158]
The `sybil` package has been removed from CRAN. Similar to the sybilSBML package that breaks the current installation intructions and lead to gapseq failures when building draft models. See https://cran.r-project.org/web/packages/sybil/index.html .

Installing sybil from the CRAN archive still works.

**Comment by jotech** [comment]
Hi @cdiener 

Thank you for reporting the issue! I was involved in the discussion with the CRAN team. The problems come from reverse dependencies to commercial solvers which are not available on the test systems and a tightened CRAN policy that does not tolerate it anymore. 

The issues are going to be addressed and meanwhile installing from github should work:
```
library(devtools)
install_github("SysBioChalmers/sybil")
```

**Comment by cdiener** [comment]
Yeah that is understandable. I think it just needs to be updated in the install instructions.

**Comment by Waschina** [comment]
Hi @cdiener 

Thanks for reporting! I adjusted the installation instructions and tested them on a fresh Debian 11 system. I am confident that the changes in the install instructions also work for Centos/Fedora/RHEL.

No changes were required in the conda install instructions.

Also, the commands to install libsbml and sybilSBML was added to the instructions for linux systems.

@jotech Instead of using devtools to fetch the latest development version of sybil, I decided to download the latest sybil version (2.2.0) from the CRAN archives. The reason is, that there might be future changes in the github sybil version, which might break something in gapseq. With the latest versions on CRAN-archives, we know that they work.

In any case, we'll hope that sybil and sybilSBML will be back on CRAN soon.
